dojo/user/validators.py

```diff
@@ -75,7 +75,7 @@ def get_help_text(self):
 
 class SymbolValidator(object):
     def validate(self, password, user=None):
-        contains_special_character = re.findall('[()[\]{}|\\`~!@#$%^&*_\-+=;:\'\",<>./?]', password)  # noqa W605
+        contains_special_character = re.findall(r'[(){}\[\]|~!@#$%^&*_\-+=;:\'",<>\./?]', password)
```

Comment:
Looks like the backtick (``` ` ```) is missing here

Reply:
I added the backtick in the [commit](https://github.com/DefectDojo/django-DefectDojo/pull/8879/commits/1e51a6351795768c064e63a76d213517375b983f)